R/rank.R

```diff
@@ -36,9 +33,13 @@
 #'   non-duplicate value will have a rank incremented by `1`, effectively
 #'   removing any gaps in the ranking.
 #'
-#' @param na_propagate A single logical specifying whether or not missing
-#'   values should be propagated. If `TRUE`, all missing values are given
-#'   the rank `NA`.
+#' @param incomplete Treatment of [incomplete][vec_detect_complete] values.
+#'
+#'   - `"rank"`: Give incomplete values a numeric rank, which is affected
```

Comment:
Using the term "rows" instead of "values" would probably be clearer?

Reply:
I stayed away from "rows" because you can rank atomic vectors too and I think this documentation needs to make sense in that case too

Reply:
I could use "observations"?